# Definition of the pid_msg
float32 p_pitch
float32 p_roll
float32 p_yaw
float32 i_pitch
float32 i_roll
float32 i_yaw
float32 d_pitch
float32 d_roll
float32 d_yaw
float32 sum_pitch
float32 sum_roll
float32 sum_yaw
